plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/query/DynamicTable.java

```diff
@@ -64,4 +66,9 @@ public boolean aggregateInProjections()
         return projections.stream()
                 .anyMatch(PinotColumnHandle::isAggregate);
     }
+
+    public Stream<PinotColumnHandle> getColumnHandlesForSelect()
```

Comment:
This class is a record. I would rename to `columnHandlesForSelect` or something for consistency with other method names.